nixos/modules/services/web-servers/apache-httpd/mediawiki.nix

```diff
@@ -318,8 +318,16 @@ in
             echo COMMIT
           ) | ${pkgs.postgresql}/bin/psql -U "${config.dbUser}" "${config.dbName}"
       fi
-      ${php}/bin/php ${mediawikiRoot}/maintenance/update.php
-    '');
+    ''}
+    ${optionalString (config.dbType == "mysql" && config.dbServer == "") ''
```

Comment:
and in fact maybe a cleaner way to express this is using if then else instead of optionalString